### PR TITLE
Feature/279908  get cdos

### PR DESCRIPTION
**Pull request by whitewaterdesign**

### CODE REVIEW

File: app/repos/cdo.js [hunk 1]
```diff
@@ -298,7 +299,10 @@ const getSummaryCdos = async (filter) => {
         ]
       }
     ],
-    order: [[sequelize.col('registration.cdo_expiry'), 'ASC']]
+    order: [
```
Comment (jbarnsley10) [inline app/repos/cdo.js:302]: Not 100% sure why  'joined_exemption_scheme' is an addition orderBy? Is it that some records don't have a cdo_expiry but an exemption_scheme date?

Comment (whitewaterdesign) [reply app/repos/cdo.js:302]: this is getting ready for the next story, although I don't think there are any without cdo_expiry at the moment.  Maybe I should just remove and replay this when I get to the story.

File: app/repos/cdo.js [hunk 2]
```diff
@@ -268,6 +278,14 @@ const getSummaryCdos = async (filter) => {
     }
   }
 
+  const order = []
+
+  if (sortKey !== undefined) {
+    order.push([sequelize.col(sortKey), sortOrder])
+  }
+
+  order.push([sequelize.col('registration.cdo_expiry'), sortOrder])
```
Comment (jbarnsley10) [inline app/repos/cdo.js:287]: So if you supply a sort order (ASC/DESC), the direction will be applied to BOTH the supplied column (if you supply a column) and to the cdo_expiry? Is it not an either/or i.e. if supplied, sort by supplied column. If not supplied, sort by cdo_expiry?

**Review by jbarnsley10 — APPROVED** [review]
Approved but I need to get my head around why you are wanting to sort on BOTH supplied column plus cdo_expiry

**Pull request merged**